packages/asc-icons/README.md

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
Could you perhaps write a little part on how asc-icons work technically? I've seen a lot of files in this PR and i'm not sure what they do and how they work...

Reply:
good point! I created a task DP-6992 to be addressed as soon as possible